### PR TITLE
add correct return type for drawflow addNode

### PR DESCRIPTION
**Pull request by joriszwart**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/jerosoler/Drawflow/blob/59a8e2cfd70423ef5877a76e85720ad99a3bc1ee/src/drawflow.js#L1184
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Review by MysteryBlokHed — CHANGES_REQUESTED** [review]
If you add a test for this change to `drawflow-tests.ts`, then the PR can be merged without needing review from a DefinitelyTyped maintainer.

<https://github.com/DefinitelyTyped/DefinitelyTyped/blob/20c94e257db31a43b954324b68411c2655f3a498/types/drawflow/drawflow-tests.ts#L1-L4>

Adding something that tests this method's return would be good, i.e.:

```typescript
// $ExpectType number | string
new Drawflow(element).addNode(...)
```

(Although valid arguments will have to be passed into `addNode`)

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/drawflow/drawflow-tests.ts
Consider running `npx dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against c05c5481a20bbd5913b7bec6812f397b791ba5e8
</p>

**Review by MysteryBlokHed — APPROVED** [review]
LGTM!

**Review by MysteryBlokHed — APPROVED** [review]

**Pull request merged**
